### ISSUE TITLE
The training PackedNLLoss is negative.

### ISSUE THREAD

**Issue by gwyanCN** [SalesforceAIResearch/uni2ts#227]
Hello, I have recently been fine-tuning my own dataset using Moirai. However, I noticed that my training loss has become negative, and the prediction loss first decreases and then increases. It seems that the model training cannot converge. Therefore, I would like to ask why my training loss has become negative and how to deal with it?

Here is my train/val loss:
![Image](https://github.com/user-attachments/assets/ef534394-b253-4784-9e6f-ec939b981afa)

**Comment by zqiao11** [comment]
Hi, the training loss is the negative log-likelihood, which can naturally take negative values. So there’s no need to worry.

Your training converges, but the issue might be overfitting. You could consider tuning hyperparameters such as the learning rate.

**Comment by gwyanCN** [comment]
> Hi, the training loss is the negative log-likelihood, which can naturally take negative values. So there’s no need to worry.
> 
> Your training converges, but the issue might be overfitting. You could consider tuning hyperparameters such as the learning rate.

Thank you for your help. I have adjusted the learning rate and other parameters, but currently, my validation NLL loss still seems abnormal, and the trend graph of the prediction doesn't look right either. Could you please tell me what might be causing this?

Here is the loss and prediction:

<img width="1139" height="885" alt="Image" src="https://github.com/user-attachments/assets/9ff82c98-18e2-44f0-af85-4cd036a1417c" />

<img width="1066" height="641" alt="Image" src="https://github.com/user-attachments/assets/1354149d-45c0-4acd-86c7-b0e982748114" />
